share/ansible_navigator/utils/catalog_collections.py

```diff
@@ -112,6 +112,29 @@ def _generate_checksum(file_path: str, relative_path: str) -> Dict:
         }
         return res
 
+    def _path_walk(self, top: Path, top_down: bool = False, follow_links: bool = False):
```

Comment:
Prefer moving this out to a util function. It doesn't seem to use the class instance anywhere and it seems more general.

Reply:
This function replaced with a glob